opm/material/fluidmatrixinteractions/PiecewiseLinearTwoPhaseMaterial.hpp

```diff
@@ -233,15 +256,7 @@ class PiecewiseLinearTwoPhaseMaterial : public TraitsT
 
         size_t segIdx = findSegmentIndex_(xValues, scalarValue(x));
 
-        Scalar x0 = xValues[segIdx];
-        Scalar x1 = xValues[segIdx + 1];
-
-        Scalar y0 = yValues[segIdx];
-        Scalar y1 = yValues[segIdx + 1];
-
-        Scalar m = (y1 - y0)/(x1 - x0);
-
-        return y0 + (x - x0)*m;
+        return(xValues,yValues,segIdx);
```

Comment:
What is this `return` statement intended to do?  From the function's return value it looks like it's supposed to construct a new `Evaluation` object, but there's no apparent constructor that will (currently) accept those arguments.  Are you missing a change to the `Evaluation` class?  I'd also recommend using a braced initialiser,
```C++
return { xValues, yValues, segIdx };
```
instead of parentheses here.

Reply:
hope fully fixed.